apps/sports-helsinki/src/domain/search/combinedSearch/adapters/CombinedSearchProvider.tsx

```diff
@@ -30,6 +31,9 @@ function useGetCombinedSearchContext(): UseGetCombinedSearchContextReturnType {
 
   const resetFormValues = () => {
     setFormValues(initialCombinedSearchFormValues);
+    for (const fallbackQueryParam of combinedSearchAdapterInputFallbacks) {
+      combinedSearchFormAdapter.searchParams.delete(fallbackQueryParam);
+    }
```

Comment:
Why is this? Is this the thing that actually clears the filte rsummary? Shouldn't the setFormValues already clear all the inputs? The filter summary section should follow the form values, but if that needs some huge refactoring still, I guess this is fine, if it would be moved in a new class-function with some API comments that developers understand why it is there for.

Reply:
Could the searchParams just be set to empty object or null instead?

Reply:
Moved to own function for clarity in 6996d2d1aa1b8314db0712955aac81434352a17d. All searchParams could perhaps be cleared but didn't do that here, instead clearing only the ones that we know can be used for backward compatibility.